test/kuttl/tests/update-openstacklightspeed/06-update-openstack-lightspeed-instance.yaml

```
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: openstack-lightspeed-apitoken-update
+  namespace: openstack-lightspeed
+type: Opaque
```

YAMLlint (1.38.0)

[error] duplication of key "type" in mapping

(key-duplicates)